hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -310,5 +358,39 @@ public void loadValidator() {
 			System.exit(0);
 		}
     	
-    }
+    }    
+
+	/**
+	 * Creates MQ connection and sets it on a JMS Component which is added to the camel context.
+	 */
+	protected void initMQ() {
+		final String methodName = LoggingUtil.getMethodName();
+		JmsComponent jmsComponent = new JmsComponent();
+    	MQQueueConnectionFactory mqQueueConnectionFactory = mqQueueConnectionFactory();
+		jmsComponent.setConnectionFactory(mqQueueConnectionFactory);
+        getContext().addComponent("jmsComponent", jmsComponent);
+		logger.info("{} - Added JMS Component to context with connection factory. {}",methodName);			
+	}
+    
+    /**
+     * Creates a MQ connection factory and sets its connection properties.
+     * 
+     * @return a {@link MQQueueConnectionFactory} factory
+     */
+    public MQQueueConnectionFactory mqQueueConnectionFactory()  {
```

Comment:
Method should be private.

Reply:
Made methods private where public not required